src/argilla/_constants.py

```diff
@@ -32,5 +32,6 @@
 OPENAI_SEPARATOR = "\n\n###\n\n"
 OPENAI_END_TOKEN = " END"
 OPENAI_WHITESPACE = " "
+OPENAI_LEGACY_MODEL = ["babbage", "davinci", "curie", "ada"]
```

Comment:
maybe we could move these constants to `src/argilla/training/openai.py`

Reply:
```suggestion
OPENAI_LEGACY_MODELS = ["babbage", "davinci", "curie", "ada"]
```

And also agree with @gabrielmbmb suggestion here